scraper.py

```diff
@@ -235,16 +243,25 @@ def scrape(output_file_name:str, no_of_articles:int, category_urls:dict, time_de
 
     # specify categories to scrape
     if params.categories != "all":
-        categories = params.categories.split(",")
-        categories = {category:ALL_CATEGORIES[category] for category in categories}
+        categories = params.categories.upper().split(",")
+        categories = {category: ALL_CATEGORIES[category] for category in categories}
     else:
         categories = ALL_CATEGORIES
 
+    articles_per_category = None
+    if params.no_of_articles > 0 and params.spread:
+        if "most_popular" in categories:
+            # most_popular only has ne page
+            articles_per_category = (params.no_of_articles-10) // (len(categories)-1)
```

Comment:
can you leave a comment that we subtract 10 because there are 10 most popular articles?